INFO - 2025-06-14 17:17:15,241 - conftest.py:21 -[conftest:start_test_and_end] - -------------接口测试开始--------------
INFO - 2025-06-14 17:17:15,241 - sendrequest.py:142 -[sendrequest:run_main] - 接口名称：用户登录
INFO - 2025-06-14 17:17:15,241 - sendrequest.py:143 -[sendrequest:run_main] - 请求地址：http://47.113.145.175:8080/ssm_web/user/login
INFO - 2025-06-14 17:17:15,247 - sendrequest.py:144 -[sendrequest:run_main] - 请求方式：POST
INFO - 2025-06-14 17:17:15,247 - sendrequest.py:145 -[sendrequest:run_main] - 测试用例名称：手机号码为空
INFO - 2025-06-14 17:17:15,247 - sendrequest.py:146 -[sendrequest:run_main] - 请求头：None
INFO - 2025-06-14 17:17:15,247 - sendrequest.py:157 -[sendrequest:run_main] - 请求参数：{'params': {'phone': '', 'password': '1234567'}}
INFO - 2025-06-14 17:17:15,362 - sendrequest.py:108 -[sendrequest:send_request] - 接口返回信息：{"success":true,"state":206,"message":"用户名密码错误","content":null}
INFO - 2025-06-14 17:17:15,363 - assertions.py:308 -[assertions:assert_result] - yaml文件预期结果：[{'contains': {'message': '用户名密码错误'}}, {'contains': {'status_code': 200}}]
INFO - 2025-06-14 17:17:15,363 - assertions.py:65 -[assertions:contains_assert] - 响应文本断言成功：预期结果【用户名密码错误】,实际结果【用户名密码错误】
ERROR - 2025-06-14 17:17:15,363 - assertions.py:41 -[assertions:contains_assert] - 接口返回码包含断言成功：预期结果【200】,实际结果【200】
INFO - 2025-06-14 17:17:15,363 - assertions.py:339 -[assertions:assert_result] - 测试成功
INFO - 2025-06-14 17:17:15,363 - conftest.py:23 -[conftest:start_test_and_end] - -------------接口测试结束--------------
